R/pkg/vignettes/sparkr-vignettes.Rmd

```diff
@@ -539,7 +539,7 @@ In the following example, we use the `longley` dataset to train a random forest
 
 ```{r}
 df <- createDataFrame(longley)
-rfModel <- spark.randomForest(df, Employed ~ ., type = "regression", numTrees = 5)
+rfModel <- spark.randomForest(df, Employed ~ ., type = "regression", maxDepth = 2, numTrees = 2)
```

Comment:
you could also do this to limit output

```{r, include=FALSE}
ops <- options()
options(max.print=40)
```

there is another example in the vignettes